emails/reset-password.pug

```diff
@@ -0,0 +1,7 @@
+extends index
+block title
+    h1 Hi, #{firstName}
+block description
+    p You have requested a password reset for your account at SpaceToStudy. Please click “reset my password” to continue.
+    a(href=`${process.env.CLIENT_URL}/activate/${activationLink}`) Reset your password
```

Comment:
The activationLink variable isn't passed to this template. `resetToken` variable is. 
Also, considering the logic of the endpoint, I don't think this is the way the link to the page for the password reset will be called.